SU-HKKU/cardinal_service_profile
# CSD-375 set up equal height cards and view items

# READY FOR REVIEW

# Summary
- Removed the flex classes to allow us to do our own thing with the heights.

# Need Review By (Date)
- soon

# Urgency
- low

# Steps to Test
1. Checkout this branch
1. `blt drupal:sync --site=cardinalservice`
1. check out the home page cards, the resource cards, and the story cards and validate they are all the same heights in the rows.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
